src/drivers/pcspkr.c

```diff
@@ -4,9 +4,9 @@
 
 void pit_phase_c2(uint32_t hz) {
   uint32_t divisor = 1193180 / hz;
-  outb(0x42, 0xB6);
-  outb(0x40, divisor & 0xFF);
-  outb(0x40, divisor >> 8);
+  outb(0x43, 0xB6);
+  outb(0x42, (uint8_t)divisor & 0xFF);
+  outb(0x42, (uint8_t)(divisor >> 8) & 0xFF);
```

Comment:
```suggestion
  outb(0x42, (uint8_t)divisor 
);
  outb(0x42, (uint8_t)(divisor >> 8));
```
There's no point in masking a `uint8_t`, is is trimmed with the conversion.